src/log.cpp

```diff
@@ -196,7 +196,16 @@ LogStream::LogStream(const std::string& file,
 
 LogStream::~LogStream()
 {
+#ifdef FUZZ
+  // When fuzzing, we don't want to output error messages. However, some
```

Comment:
Why do we need to suppress error messages? Is it a speed thing? I thought fuzzing engines look for program crashes -- the output shouldn't matter to them.

Reply:
libfuzzer prints the program's stdout/stderr to a terminal by default, and it makes reading libfuzzer's output almost impossible. We can change the behavior `-close_fd_mask` flags of libfuzzer( https://llvm.org/docs/LibFuzzer.html). But I think suppressing in bpftrace is better. And maybe another fuzzer behaves like libfuzzer.

Reply:
Ah, got it.